cdc/sink/ddlsink/mysql/async_ddl.go

```diff
@@ -92,9 +96,23 @@ func (m *DDLSink) asyncExecDDL(ctx context.Context, ddl *model.DDLEvent) error {
 	}
 }
 
+func (m *DDLSink) needWaitAsyncExecDone(ddl *model.DDLEvent) bool {
```

Comment:
It looks this method only use `ddl.Type`, so just pass the ddl type as the parameter, no need to pass the whole DDL, this makes write unit test easier.

Reply:
can you add a unit test to cover this method ?